R/sqrt_OU_covariance.R

```diff
@@ -91,6 +91,14 @@ sqrt_OU_covariance <- function(tree, alpha=0, root.model = c("OUfixedRoot", "OUr
         tre <- transf.branch.lengths(tree, model=root.model, parameters=list(alpha=alpha), check.pruningwise=F)$tree
 	if(normalize.tree.height){
 		tre <- normalize_tree(tre)
+		coe = 2*alpha # good for "OUrandomRoot"
+		if (root.model == "OUfixedRoot"){
+			# below: assumes tree already in pruningwise = post order
+			treeheight = pruningwise.distFromRoot(tre)[1] # Distance taxon 1 to root: because ultrametric tree
+			coe = (2*alpha)/(1-exp(-2*alpha*treeheight))
```

Comment:
It would be best to 
- remove `normalize_tree`
- divide all branch lengths be 2 alpha, including the root edge length.
nothing else.

`normalize_tree` messes up with the tree height. Don't normalize, and you don't need to re-calculate the tree height.